src/main/app/src/app/formulieren/formio-wrapper/formio-wrapper.component.less

```diff
@@ -0,0 +1,6 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Dimpact
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+@import "../../../../node_modules/bootstrap/dist/css/bootstrap.css";
```

Comment:
The Angular CSS and schema are also loaded from node modules; not my favorite solution. Maybe leave out the relative path?

`@import "~bootstrap/dist/css/bootstrap.css";`

Reply:
IntelliJ does not like using this:
![image](https://github.com/user-attachments/assets/e2261507-85da-42a2-a157-a017e8b47dbb)